recipe/meta.yaml

```diff
@@ -34,17 +32,18 @@ test:
     - coveralls --help
 
 about:
-  home: https://github.com/coagulant/coveralls-python
+  home: {{ git_url }}
   license: MIT
   license_family: MIT
-  license_file: LICENCE  # Yes it is a typo, will be fixed in the next release
+  license_file: LICENSE.txt
   summary: Show coverage stats online via coveralls.io
   description: |
-    Coveralls.io is service to publish your coverage stats online with a lot
-    of nice features. This package provides seamless integration with
-    coverage.py in your python projects.
-  dev_url: https://github.com/coagulant/coveralls-python
+    coveralls.io is a service for publishing your coverage stats online. This
```

Comment:
I noted that you probably copied the info here to the README file.
We only edit the recipe and use `conda-smithy`, a conda-forge tool, to "re-render" the feedstock.

For more info see: https://conda-forge.org/docs/conda_smithy.html

Reply:
Do I need to run smithy, or is that done automagically?

Reply:
No need to run it here unless you want to see how it works. We have a bot that run it periodically, but we do recommend people to always re-render to give the bot a break sometimes :wink: